lib/offsite_payments/integrations/hi_trust.rb

```diff
@@ -31,7 +31,7 @@ def initialize(order, account, options = {})
           add_field('Type', 'Auth')
 
           # Capture the payment right away
-          add_field('depositflag', options[:deposit_flag] || '0')
+          add_field('depositflag', '1')
```

Comment:
Should we still leave the option but default to `'1'` instead of `'0'`? Or did hitrust say this should _always_ be `'1'`?